releases.md

```diff
@@ -0,0 +1,201 @@
+
+---
+
+### Buidling coreboot image
+
+In order to build coreboot image, follow the steps below:
+
+1. Clone the coreboot repository:
+
+   ```
+   git clone git@github.com:3mdeb/coreboot.git -b talos_2_support
+   # or HTTPS alternatively
+   git clone https://github.com/3mdeb/coreboot.git -b talos_2_support
+   ```
+   `talos_2_support` is the main development branch for Talos II support.
+
+2. Get the submodules:
+
+   ```
+   cd coreboot
+   git submodule update --init --checkout
+   ```
+
+3. Start docker container (assuming you are already in coreboot root
+   directory):
+
+   ```
+   docker run --rm -it -v $PWD:/home/coreboot/coreboot -w /home/coreboot/coreboot coreboot/coreboot-sdk:65718760fa /bin/bash
+   ```
+
+4. When inside of the container, configure the build for Talos II:
+
+   ```
+   make menuconfig
+   ```
+
+   * Navigate to the **Mainboard** submenu.
+   * As a **Mainboard vendor** select `Raptor Computing Systems`
+   * If it wasn't selected autmatically, as **Mainboard model** select `Talos II`
+   * In the **ROM chip size** option select `512 KB`
+   * Save the configuration and exit.
+
+   ![make menuconfig](images/cb_menuconfig.png)
+
+5. Start the build process of coreboot inside the container:
+
+   ```
+   make
+   ```
+
+---
+
+### Running the coreboot on Talos II
+
+1. At the end of build process you should see `Built raptor-cs/talos-2 (Talos II)`.
+   Copy the result binary from `<coreboot_dir>/build/coreboot.rom.signed.ecc` to the BMC
+   (assuming in the coreboot root directory):
+
+   ```
+   scp build/coreboot.rom.signed.ecc root@<BMC_IP>:/tmp
+   ```
+   > If that file is not present, use `coreboot.rom` instead
+
+2. Backup the HBB partition (for faster later recovery) by invoking this
+   command on BMC:
+
+   ```
+   pflash -P HBB -r /tmp/hbb.bin
+   ```
+
+3. Flash the binary by replacing HBB partition (execute from BMC):
+
+   ```
+   pflash -e -P HBB -p /tmp/coreboot.signed.ecc
+   ```
+   > Again, if that file is not present, use `coreboot.rom` instead
+
+   Answer yes to the prompt and wait for the process to finish.
+
+4. Log into the BMC GUI again at https://\<BMC_IP\>/. Enter the Server power
+   operations (https://\<BMC_IP\>/#/server-control/power-operations) and invoke
+   warm reboot. Then move to Serial over LAN remote console
+   (https://\<BMC_IP\>/#/server-control/remote-console)
+
+   Wait for a while until coreboot shows up:
+
+   [![asciicast](https://asciinema.org/a/OTEPFRHlasyXQI2eRBLso0AB0.svg)](https://asciinema.org/a/OTEPFRHlasyXQI2eRBLso0AB0)
+
+5. Enjoy the coreboot running on Talos II.
+
+> **Optional:** In order to recovery the platform quickly to healthy state, flash
+> the HBB partition back with: \
+> `pflash -e -P HBB -p /tmp/hbb.bin`
+
+### Hardware configuration
+
+Configuration with a single IBM POWER9 64bit CPU is supported. \
+Dual CPU setup not supported currenctly.
+
+Following RAM configurations were tested and are proved to be properly initialized.
```

Comment:
Maybe add a note that all 3 major DRAM vendors were used, i.e. Samsung, Micron, Hynix